Doc/library/asyncio-task.rst

```diff
@@ -128,17 +143,15 @@ using the :meth:`sleep` function::
     import datetime
 
     async def display_date(loop):
+        loop = asyncio.get_running_loop()
         end_time = loop.time() + 5.0
         while True:
             print(datetime.datetime.now())
             if (loop.time() + 1.0) >= end_time:
                 break
             await asyncio.sleep(1)
```

Comment:
this is a bit more concise

```python
         while (loop.time() + 1.0) < end_time:
              print(datetime.datetime.now())
              await asyncio.sleep(1)
```

Reply:
We can replace this while loop with `for _ in range(5)` and the example will only become clearer.

We'll have a separate pass over asyncio docs before 3.7 is released.  We'll try to come up with better examples and improve the current ones. So for now, I'd keep this snippet as is.